lib/Extension/LanguageServerRename/Handler/FileRenameHandler.php

```diff
@@ -39,14 +45,42 @@ public function methods(): array
     public function willRenameFiles(RenameFilesParams $params): Promise
     {
         return call(function () use ($params) {
-            $workspaceEdits = LocatedTextEditsMap::create();
-            foreach ($params->files as $rename) {
-                assert($rename instanceof FileRename);
+            $count = 0;
+            $documentChanges = [];
+            try {
+                foreach ($params->files as $rename) {
+                    $locatedEditMap = LocatedTextEditsMap::create();
+                    assert($rename instanceof FileRename);
```

Comment:
shouldn't need `assert` here, `$params->files` is a generic.

Reply:
I looked how `RenameFilesParams` instance is created - indeed there is rather no way for which this assertion could be needed.